app/models/food.rb

```diff
@@ -0,0 +1,12 @@
+class Food < ApplicationRecord
+  belongs_to :user
+  has_many :recipe_foods, dependent: :destroy
+
+  def food_quantity
+    quantity = 0
+    recipe_foods.each do |food|
+      quantity += food.quantity
+    end
```

Comment:
- [ ] The controllers should be concerned with HTTP verbs. (thin controller rule) By that, I mean that a controller’s responsibility should be focused on web requests. Other calculations should be handled on the model.

Reply:
@sinansevgi Hey man, I don't really understand this change. The calculations here are inside the model for food. Isn't this what you are suggesting?

Reply:
You are calculating quantity, you should do it in your model.